chart/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: trow
+description: A Helm chart for Trow Registry
+
+type: application
+version: 0.1.0
+appVersion: 0.3.0
```

Comment:
Yeah, I guess this should be 0.2.0 for the moment. I do need to do a new release, but I think it will be next week.

Apologies - I know I told you to put 0.3.0 before.